ServerCore/Pages/Teams/Play.cshtml

```diff
@@ -54,15 +54,19 @@
             <td>
                 @if (item.Puzzle.CustomURL != null)
                 {
-                    <a href="@ServerCore.Helpers.PuzzleHelper.GetFormattedUrl(item.Puzzle, Model.Event.ID)" target="_blank">@item.Puzzle.Name</a>
+                <a href="@ServerCore.Helpers.PuzzleHelper.GetFormattedUrl(item.Puzzle, Model.Event.ID)" target="_blank">@item.Puzzle.Name</a>
                 }
                 else if (item.Content != null)
                 {
-                    @Html.ActionLink(item.Puzzle.Name, "Index", "Files", new { eventId = Model.Event.ID, filename = item.Content.ShortName }, new { target = "_blank" })
+                @Html.ActionLink(item.Puzzle.Name, "Index", "Files", new { eventId = Model.Event.ID, filename = item.Content.ShortName }, new { target = "_blank" })
                 }
                 else
                 {
-                    @Html.DisplayFor(modelItem => item.Puzzle.Name)
+                @if (item.Puzzle.Errata != null)
```

Comment:
I'm not sure if it would be null if somebody adds and removes errata. Maybe String.IsNullOrWhitespace?

Reply:
Good idea, done